packages/helpers/src/error/index.ts

```diff
@@ -0,0 +1,22 @@
+import { ShopwareError } from "@shopware-pwa/commons/interfaces/errors/ApiError";
+
+/**
+ * Get the messages from the API response (array of ShopwareErrors)
+ */
+export function getMessagesFromErrorsArray(
+  errors: ShopwareError[]
+): String[] | undefined {
+  if (!errors || !Array.isArray(errors) || !errors.length) {
+    return;
+  }
```

Comment:
would be nicer to always return an array :)
+ return primitive string
```suggestion
export function getMessagesFromErrorsArray(
  errors: ShopwareError[]
): string[] {
  if (!errors?.length || !Array.isArray(errors)) {
    return [];
  }
```

Reply:
sure! it's more elegant :+1: